src/exported-image-egl.cpp

```diff
@@ -52,13 +52,3 @@ wpe_fdo_egl_exported_image_get_egl_image(struct wpe_fdo_egl_exported_image* imag
 }
 
 }
-
-void
-wpe_fdo_egl_exported_image_destroy(struct wpe_fdo_egl_exported_image* image)
-{
-    assert(image->eglImage);
-    WS::Instance::singleton().destroyImage(image->eglImage);
-    wl_list_remove(&image->bufferDestroyListener.link);
```

Comment:
This is no longer needed?

Reply:
It's moved into the deleteImage() method on the ClientBundleEGL class. So everything still gets destroyed, just the code was localized to that class.

Reply:
I meant the wl_list_remove specifically, github doesn't seem allow to select the lines in the review.

Reply:
I think it's not needed. At this point the resource is already destroyed or is being destroyed, so removing that listener doesn't do much.